package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "bluebird": "~2.9.25",
-    "fuel-auth": "~1.0.0",
+    "fuel-auth": "2.1.0",
```

Comment:
I didn't realize this library hasn't started using `fuel-auth` 2.0 before this. We may want to take a closer look at what we should do to make this jump.